mixins/test/label-mixin.test.js

```diff
@@ -172,35 +162,31 @@ describe('LabelledMixin', () => {
 		});
 
 		it('updates label when labelling element is replaced', async() => {
+			elem = await fixture(customElemFixture);
 			const labelledElem = elem.querySelector('[labelled-by="label2"]');
 			const labelElem = elem.querySelector('#label2');
 			const newLabelElem = document.createElement(labelTag);
 			newLabelElem.id = 'label2';
 			newLabelElem.text = 'new label value';
 			labelElem.parentNode.replaceChild(newLabelElem, labelElem);
-			await nextFrame();
+			await oneEvent(labelledElem, 'd2l-labelled-mixin-label-elem-change');
 			expect(labelledElem.shadowRoot.querySelector('input').getAttribute('aria-label')).to.equal('new label value');
 		});
 
-		it('does not explode if invalid id reference provided', async() => {
-			const labelledElem = elem.querySelector('[labelled-by="invalidlabel"]');
-			expect(labelledElem.shadowRoot.querySelector('input').hasAttribute('aria-label')).to.equal(false);
-		});
```

Comment:
I had to delete the tests that would cause exceptions as there's no way to catch them. :(